pydatastructs/graphs/tests/test_algorithms.py

```diff
@@ -292,6 +292,8 @@ def _test_shortest_paths(ds, algorithm):
     _test_shortest_paths("Matrix", 'bellman_ford')
     _test_shortest_paths("List", 'dijkstra')
     _test_shortest_paths("Matrix", 'dijkstra')
+    _test_shortest_paths("List", 'floyd_warshall')
+    _test_shortest_paths("Matrix", 'floyd_warshall')
```

Comment:
`floyd_warshall` is a pair wise shortest path algorithm so tests of `dijkstra` and `bellman_ford` may not work. As can be seen, API for both are different.